plugins/woocommerce/templates/order/attribution-data-fields.php

```diff
@@ -6,7 +6,7 @@
  *
  * @see     Automattic\WooCommerce\Internal\Orders\OrderAttributionController
  * @package WooCommerce\Templates
- * @version 8.4.0
+ * @version 8.4.1
```

Comment:
Changed since the PR was added.
```suggestion
 * @version 8.6.0
```

Reply:
I think this change should fix the tests (this was the only one still failing when I reviewed).